sylt-parser/src/parser.rs

```diff
@@ -310,6 +310,21 @@ macro_rules! expect {
     };
 }
 
+/// Eat until any one of the specified tokens or EOF.
+#[macro_export]
+macro_rules! until {
+    ($ctx:expr, $( $token:pat )|+ ) => {
+        {
+            let mut ctx = $ctx;
+            while !matches!(ctx.token(), T::EOF | $( $token )|*) {
+                ctx = ctx.skip(1);
+            }
+            ctx
+        }
+    };
+}
+
+
```

Comment:
Couldn't this be a regular function?

Reply:
Yes - it could be.
BUT:
 - Not if you want to allow multiple tokens passed into the `matches` macro - no use of it right now so I might be over zealous
  - I also felt it was more inline with how "expect" works
  
It's also fair to mention `skip_if` which functions similarly and is infact a function.

Reply:
While on the subject on this macro, maybe `skip_until` instead?